### PR TITLE
Create new columns in sample_summary.tsv

### PR DESCRIPTION
**Pull request by jessicarowell**
Created 3 new columns:
opxv_reads_1 -> filepath to seqtk output, reads file 1
opxv_reads_2 -> filepath to seqtk output, reads file 2
final_assembly -> filepath to final assembly

Test command:
`nextflow run main.nf --indir /scicomp/groups-pure/Projects/scbs_mpob/JRowell_ick4/mpox_testing/paired/ --outdir /scicomp/groups-pure/Projects/scbs_mpob/JRowell_ick4/mpox_testing/results/ --project_name test -config /scicomp/reference/nextflow/configs/cdc-dev.config -profile singularity,rosalind`

## PR checklist

- [ x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mpxvassembly/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/mpxvassembly _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by mikeyweigand** [comment]
Make sure outputs are full paths to fasta and fastq, even if user input is a relative path.

**Comment by jessicarowell** [comment]
I've added a line to convert relative to absolute paths for outdir

**Comment by jessicarowell** [comment]
Right now this code only outputs a path for final_assembly if there is a `<sample>.final.fa` file.  If there are only `<sample>.draft.fa`  assemblies, these are ignored and the column is left blank. Is this the expected behavior or should I modify?

File: modules/local/summarize_qc.nf
Comment (jessicarowell) [inline modules/local/summarize_qc.nf:1]: Whoops.  Sorry about that...blaming it on Friday. Thanks for fixing it.

**Comment by mikeyweigand** [comment]
> @mikeyweigand Right now this code only outputs a path for final_assembly if there is a `<sample>.final.fa` file. If there are only `<sample>.draft.fa` assemblies, these are ignored and the column is left blank. Is this the expected behavior or should I modify?

Agree this makes the most sense, current 'ask' is to streamline capture of necessary inputs for TOSTADAS, to which you wouldn't send a draft contig fasta.

**Pull request merged**